src/openforms/registrations/contrib/objects_api/handlers/v1.py

```diff
@@ -0,0 +1,109 @@
+"""
+Helpers for the template based Objects API registration handler.
+"""
+
+from collections.abc import Sequence
+from datetime import datetime
+from decimal import Decimal
+from typing import Literal, TypedDict
+
+from openforms.authentication.service import AuthAttribute
+from openforms.contrib.objects_api.rendering import render_to_json
+from openforms.submissions.models import Submission
+from openforms.typing import JSONObject
+from openforms.variables.utils import get_variables_for_context
+
+from ..registration_variables import get_cosign_value
+
+
+class PaymentContextData(TypedDict):
```

Comment:
Wouldn't be better if we had these TypeDicts in a separate file?

Reply:
I didn't think the possibility that someone will make use of them, let's keep it as is then, I agree.